src/main/java/nextstep/subway/line/dto/LineResponse.java

```diff
@@ -26,8 +27,12 @@ public LineResponse(Long id, String name, String color, List<StationResponse> st
         this.modifiedDate = modifiedDate;
     }
 
-    public static LineResponse of(Line line, List<StationResponse> stations) {
-        return new LineResponse(line.getId(), line.getName(), line.getColor(), stations, line.getCreatedDate(), line.getModifiedDate());
+    public static LineResponse of(Line line) {
+        List<StationResponse> stations = line.sections().orderedStations().stream()
```

Comment:
```suggestion
        List<StationResponse> stations = line.orderedStations().stream()
```
```
최소한의 지식 원칙(The Principle of Least Knowledge) 을 강조한며 모듈은 자신이 조작하는 객체의 속사정을 몰라야 한다는 법칙
```
`Sections`의 `orderedStations` 메서드를 추가적으로 호출해야할까요?
위와 같은 경우 `Sections`의 존재를 몰라도 될것같은데 어떻게 생각하시나요?  :thinking: 


[디미터 법칙](https://prohannah.tistory.com/204)